docs/index.rst

```diff
@@ -79,6 +78,15 @@ Seismology and geodesy
 Basemaps
 ~~~~~~~~
 
+.. cssclass:: gmtgallery
+
+-  .. image:: _images/7dd36e07258e1015a0673dea4e472db1.png
+      :target: gallery/maps/gadm.html
+      :width: 80%
+      :align: center
+
+   :doc:`gallery/maps/gadm`
```

Comment:
You don't need to open a separate PR #29, just need to update the following lines:
```suggestion
.. grid:: 2 3 3 4

    .. grid-item-card:: :doc:`gallery/maps/gadm`
       :padding: 1
       :link-type: doc
       :link: gallery/maps/gadm

       .. image:: _images/7dd36e07258e1015a0673dea4e472db1.png
```